recipe/meta.yaml

```diff
@@ -6,36 +6,47 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 347f23769679aaf7efa73e5655270fcda8ca56be65eb84a4a21d143989541045
+  # Source on pypi is missing files required for tests.
```

Comment:
I'm curious, which files are missing? I see the tests in the sdist: https://inspector.pypi.io/project/gdown/4.7.1/packages/cc/36/3cc5d71e95cd65db5db8b06f9047257b417330873977626aa416076cca4f/gdown-4.7.1.tar.gz/

Reply:
The tests/data directory isn't present.

Reply:
Ah ok. Can you edit the comment to say which files/folders are missing please?